#! /bin/sh /usr/share/dpatch/dpatch-run
## 02jheretic.ded-debian-hook.dpatch by  <jamie_jones_au@yahoo.com.au>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This adds the jheretic-debian.ded that is responsible for ensuring
## DP: the jpacks are loaded, and that they are disabled when they
## DP: conflict with any mods or pwads.

@DPATCH@
diff -urNad deng-1.8.9+1.9.0beta2/Defs/jHeretic/jHeretic.ded /tmp/dpep.UL7NQo/deng-1.8.9+1.9.0beta2/Defs/jHeretic/jHeretic.ded
--- deng-1.8.9+1.9.0beta2/Defs/jHeretic/jHeretic.ded	2004-01-20 06:25:35.000000000 +1100
+++ /tmp/dpep.UL7NQo/deng-1.8.9+1.9.0beta2/Defs/jHeretic/jHeretic.ded	2005-08-14 02:20:47.938045728 +1000
@@ -25,6 +25,7 @@
 Include "Text.ded";
 Include "Values.ded";
 Include "Models.ded";
+Include "jheretic-debian.ded";
 Include "User.ded";
 
 IncludeIf Not -noauto "Auto\*.ded";
